layouts/robots.txt

```diff
@@ -1,5 +1,4 @@
 User-agent: *
-{{- /* baseURL is a template.URL, so we need to convert it before we compare it.  */ -}}
-{{- if ne (string .Site.BaseURL) "https://www.pulumi.com/" }}
+{{- if eq hugo.Environment "production" }}
```

Comment:
This should be  `ne`, since we want _not_ to be crawled/indexed _unless_ production.

Reply:
Fixed, thanks.